changelog.txt

```diff
@@ -12,6 +12,7 @@
 * Add - Onboarding flows on the admin WooCommerce > Subscriptions screen for stores with no subscriptions yet.
 * Fix - Fatal error on thank you page for deleted orders.
 * Add - Error messages when dispute evidence exceeds Stripe limits.
+* Update - Get and update merchant branding for in-person payment receipts via the Settings API endpoint.
```

Comment:
`Add - Support business account branding settings` would better describe this PR. (same for other changelog entry in readme)